src/main/java/com/marklogic/appdeployer/command/PayloadPropertyFilter.java

```diff
@@ -0,0 +1,55 @@
+package com.marklogic.appdeployer.command;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.marklogic.mgmt.PayloadParser;
+import com.marklogic.rest.util.Fragment;
+import org.jdom2.Element;
+import org.jdom2.output.XMLOutputter;
+
+public class PayloadPropertyFilter {
```

Comment:
I think this functionality can go right into PayloadParser, which doesn't do that much yet.

Reply:
agreed